docs/variants/novacustom_ns5x_adl/recovery.md

```diff
@@ -0,0 +1,187 @@
+# Recovery
+
+## Introduction
+
+This document describes the recovery procedure of laptops flashed with bad
+firmware (e.g. "bricked" devices), as well as external flashing in general,
+which can sometimes be needed when switching between different firmware vendors.
+
+This procedure can be used to restore both Dasharo and previous Insyde firmware.
+
+## Recovery
+
+### BIOS recovery
+
+You will need:
+
+- a CH341a programmer with 3.3V support
+- a WSON-8 probe
+
+A complete set containing everything you need is available from
+[our shop](https://shop.3mdeb.com/shop/modules/ch341a-flash-bios-usb-programmer-kit-soic8-sop8/).
+
+#### Prerequisites
+
+- Flashrom installed on a Linux host
+- BIOS image file to flash
+
+#### Recovery steps
+
+1. Attach the WSON probe to the programmer. Take care to align pin 1 indicated
+   on the probe's breakout board with pin 1 on the programmer:
+
+    ![](/images/ch341a_rec/ch341a_v17_with_breakout.webp)
```

Comment:
The images don't render well. They would look better if two of them were horizontal.
![image](https://github.com/user-attachments/assets/2fe276d4-7313-4891-a567-994550268a3e)